app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/screen/exam/QuestionExamFragment.kt

```diff
@@ -0,0 +1,113 @@
+package com.nguyennhatminh614.motobikedriverlicenseapp.screen.exam
+
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.view.isVisible
+import com.nguyennhatminh614.motobikedriverlicenseapp.R
+import com.nguyennhatminh614.motobikedriverlicenseapp.data.model.ExamState
+import com.nguyennhatminh614.motobikedriverlicenseapp.data.model.QuestionOptions
+import com.nguyennhatminh614.motobikedriverlicenseapp.data.model.Questions
+import com.nguyennhatminh614.motobikedriverlicenseapp.databinding.FragmentQuestionLayoutBinding
+import com.nguyennhatminh614.motobikedriverlicenseapp.screen.appadapter.QuestionOptionAdapter
+import com.nguyennhatminh614.motobikedriverlicenseapp.utils.base.BaseFragment
+import com.nguyennhatminh614.motobikedriverlicenseapp.utils.constant.AppConstant
+import com.nguyennhatminh614.motobikedriverlicenseapp.utils.extensions.loadGlideImageFromUrl
+import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+
+class QuestionExamFragment
+    : BaseFragment<FragmentQuestionLayoutBinding>(FragmentQuestionLayoutBinding::inflate) {
+
+    private var questionsPosition = AppConstant.NONE_POSITION
+    private var questions: Questions? = null
+    private var lastSelectedPosition = AppConstant.NONE_POSITION
+
+    override val viewModel by sharedViewModel<ExamViewModel>()
+
+    private val questionOptionAdapter by lazy { QuestionOptionAdapter() }
+
+    override fun initData() {
+        questions?.let {
```

Comment:
```suggestion
        questions?.let { question ->
```
Chú ý dùng nhiều scope thì phải khai báo rõ it ra nhé